#135 add logging to account matching

alensiljak opened the pull request:
Other than adding the logger.error, I added loguru for structured log output. 
Perhaps a regular print() message would also work here, to tell the user what the problem is.

redstreet commented:
Sounds good! Would you mind please rebasing and getting the checks to pass, and I'll merge once done. Note we use conventional commits, see README.md for the link. Thanks!

alensiljak commented:
Yes, I added the dependency to the pyproject while the original ones are likely in setup. I need to see what your build process is. I converted my projects to work with UV and never looked back.
Will look at this when I'm back, continuing with the import setup.

redstreet commented:
I'm not opposed to switching to UV. This is a pretty simple project from a build perspective anyway. PyPI build is [here](https://github.com/redstreet/beancount_reds_importers/blob/main/.github/workflows/pythonpublish.yml) if you want to create a PR for that?

alensiljak commented:
This one can now be merged as it is a relatively simple addition to the pyproject and a log output in case of an error.
Further messages (warn/error) may be added the same way in the future. (separately)

alensiljak commented:
BTW, running 
```sh
uv run ruff format --check
```
results in
```
87 files already formatted
```
locally so I'm not sure why the build is failing.

alensiljak commented:
Ah, I see. It's not ruff but isort.
